election-config/Ballerina.toml

```diff
@@ -0,0 +1,19 @@
+[package]
+org = "user"
```

Comment:
Let's have a more specific name instead of using a generic name as the organization name